srml/council/src/motions.rs

```diff
@@ -42,9 +42,30 @@ pub trait Trait: CouncilTrait {
 /// Origin for the council module.
 #[derive(PartialEq, Eq, Clone)]
 #[cfg_attr(feature = "std", derive(Debug))]
-pub enum Origin {
-	/// It has been condoned by a given number of council members.
-	Members(u32),
+pub enum RawOrigin<AccountId> {
+	/// It has been condoned by a given number of council members from a given total.
```

Comment:
```suggestion
	/// It has been condoned by a given number of council members where a given threshold applies.
```

Reply:
Incorrect. The original was correct though.